src/main/java/net/syscon/elite/api/model/PrisonerDetail.java

```diff
@@ -24,73 +21,108 @@
 @AllArgsConstructor
 @NoArgsConstructor
 @EqualsAndHashCode
+@Data
 public class PrisonerDetail {
     @JsonIgnore
     private Map<String, Object> additionalProperties;
 
+    @ApiModelProperty(required = true, value = "The prisoner's unique offender number (aka NOMS Number in the UK).")
```

Comment:
tend to put examples on these now..

Reply:
I've added examples for the religion, religionCode, ethnicity and ethnicityCode properties.